rumbl/priv/repo/backend_seeds.exs

```diff
@@ -0,0 +1,4 @@
+alias Rumbl.Repo
+alias Rumbl.User
```

Comment:
@shouston3 that _is_ a nice pattern. 👍 
but I _agree_ that keeping it "_faithful_" to the book at this point is a good plan. 😉